app/cli/cmd/attestation_init.go

```diff
@@ -37,14 +38,14 @@ func newAttestationInitCmd() *cobra.Command {
 		Annotations: map[string]string{
 			useAPIToken: "true",
 		},
-		PreRunE: func(cmd *cobra.Command, args []string) error {
+		PreRunE: func(cmd *cobra.Command, _ []string) error {
 			if workflowName == "" {
-				return errors.New("workflow name is required, set it via --name flag")
+				return errors.New("workflow name is required, set it via --workflow flag")
```

Comment:
I got some feedback from @juamedgod that `--name` was confusing since it seemed the name  of the attestation itself. 

So I have changed it to `workflow` while keeping the existing `--name` as deprecated